### ISSUE TITLE
Medallia 2.0 a11y - [Major] Content, scrollbar should be accessible at 300% zoom

### ISSUE THREAD

**Issue by aubreyarcangel** [department-of-veterans-affairs/va.gov-team#78313]
## Issue Description
On desktop, when the browser is zoomed to 300% or more, content and the vertical scrollbar cut off. The user would have to scroll in two dimensions to view all content.

Looking at the code, it appears that the modal with id "survey-wrapper" is programatically set to a specific width - 450px. I believe that changing "width: 450px" to "max-width: 450px", and adding "overflow: auto" as CSS propreties to #survey-wrapper will fix this issue, but a developer should try it to be sure!

Also note that I'm viewing this on desktop, NOT mobile - I don't have any work-assigned mobile GFE. This needs to be reviewed on a mobile device to check for cutoff and scrolling issues.

This issue was identified during the [CAIA Medallia 2.0 a11y audit
](https://docs.google.com/spreadsheets/d/1Z_xXH2WnyP0YWNCJFKs0duwgSIug9VNEmzWR9RzxSYM/)

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by ATMiddleton** [comment]
https://app.slack.com/client/T03FECE8V/C05NUG2QB9S?selected_team_id=T1ZD0UBMZ

**Comment by aubreyarcangel** [comment]
duplicate of https://app.zenhub.com/workspaces/vagov-veteran-support-62cdd9546ec1530018209672/issues/gh/department-of-veterans-affairs/va.gov-team/78595
